native/c/zjb.cpp

```diff
@@ -451,7 +451,10 @@ int zjb_view_by_jobid(ZJB *zjb, string jobid, ZJob &job)
 
   STATJQTR *PTR64 jobInfoNext = jobInfo;
 
-  cout << "entries " << entries << endl;
+  if (silent == false)
```

Comment:
this was my bad, i mistakenly left a debug statement and this `cout` and this can be deleted